weldx/core.py

```diff
@@ -909,7 +909,8 @@ def __init__(
         Create a `GenericSeries` representing a translation with 3 m/s in x-direction
         starting at point ``[0, 2 ,2] cm``
 
-        >>> from weldx import GenericSeries, Q_
+        >>> from weldx.core import GenericSeries
+        >>> from weldx.constants import Q_
```

Comment:
is there a bug/import error when we import from the main `weldx` namespace in the doctests?

Reply:
See my comment above, no there isn't.